migrations/20231210123554_init.sql

```diff
@@ -0,0 +1,314 @@
+    content     TEXT NOT NULL,
+    -------------------------------------------------------
+    created_at  TIMESTAMP NOT NULL DEFAULT now(),
+    edited_at   TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at  TIMESTAMP,
+    -------------------------------------------------------
+    event_id    UUID,
+    author_id   UUID,
+    task_id     UUID,
+    -------------------------------------------------------
+    FOREIGN KEY (event_id) REFERENCES event (id),
+    FOREIGN KEY (author_id) REFERENCES "user" (id),
+    FOREIGN KEY (task_id) REFERENCES task (id),
+    -------------------------------------------------------
+    CONSTRAINT check_comment_single_relation_only
+        CHECK (
+            (CASE WHEN event_id IS NULL THEN 0 ELSE 1 END
+             + CASE WHEN task_id IS NULL THEN 0 ELSE 1 END
+            ) = 1),
+    CONSTRAINT check_comment_created_at_lte_edited_at
+        CHECK (edited_at >= created_at)
+
+);
+
+
+CREATE TABLE associated_company
+(
+    type        association NOT NULL,
+    -------------------------------------------------------
+    created_at  TIMESTAMP NOT NULL DEFAULT now(),
+    edited_at   TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at  TIMESTAMP,
+    -------------------------------------------------------
+    company_id  UUID,
+    event_id    UUID,
+    -------------------------------------------------------
+    FOREIGN KEY (company_id) REFERENCES company (id),
+    FOREIGN KEY (event_id) REFERENCES event (id),
+    -------------------------------------------------------
+    CONSTRAINT check_associated_company_created_at_lte_edited_at
+        CHECK (edited_at >= created_at)
+);
+
+
+CREATE TABLE adress
+(
+    id          UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    -------------------------------------------------------
+    country     VARCHAR(255) NOT NULL,
+    region      VARCHAR(255) NOT NULL,
+    city        VARCHAR(255) NOT NULL,
+    street      VARCHAR(255) NOT NULL,
+    number      VARCHAR(255) NOT NULL,
+    postal_code VARCHAR(255) NOT NULL,
+    -------------------------------------------------------
+    company_id  UUID,
+    -------------------------------------------------------
+    FOREIGN KEY (company_id) REFERENCES company (id),
+    -------------------------------------------------------
+    CONSTRAINT check_company_address_country_len
+        CHECK (char_length(country) >= 1),
+    CONSTRAINT check_company_address_region_len
+        CHECK (char_length(region) >= 1),
+    CONSTRAINT check_company_address_city_len
+        CHECK (char_length(city) >= 1),
+    CONSTRAINT check_company_address_street_len
+        CHECK (char_length(street) >= 1),
+    CONSTRAINT check_company_address_address_number_len
+        CHECK (char_length(number) >= 1),
+    CONSTRAINT check_company_address_postal_code_len
+        CHECK (char_length(postal_code) >= 1)
+);
+
+
+CREATE TABLE employment
+(
+    hourly_rage FLOAT NOT NULL,
```

Comment:
While I do enjoy the idea of a worker getting an allotted slot for pure rage 😆 , it might be better to rename this. I think you were going for hourly_wage, or hourly_rate.